pkg/email/system_intake_create_grb_reviewer.go

```diff
@@ -0,0 +1,76 @@
+package email
+
+import (
+	"bytes"
+	"context"
+	"errors"
+	"path"
+
+	"github.com/google/uuid"
+
+	"github.com/cmsgov/easi-app/pkg/apperrors"
+	"github.com/cmsgov/easi-app/pkg/models"
+)
+
+type systemIntakeCreateGRBReviewerParameters struct {
+	ProjectTitle             string
+	RequesterName            string
+	SystemIntakeAdminLink    string
+	ITGovernanceInboxAddress string
+}
+
+func (sie systemIntakeEmails) createGRBReviewerBody(
+	systemIntakeID uuid.UUID,
+	projectTitle string,
+	requesterName string,
+) (string, error) {
+	adminPath := path.Join("governance-review-team", systemIntakeID.String(), "grb-review")
+
+	data := systemIntakeCreateGRBReviewerParameters{
+		ProjectTitle:             projectTitle,
+		RequesterName:            requesterName,
+		SystemIntakeAdminLink:    sie.client.urlFromPath(adminPath),
+		ITGovernanceInboxAddress: sie.client.config.GRTEmail.String(),
+	}
+
+	var b bytes.Buffer
+	if sie.client.templates.systemIntakeCreateGRBReviewer == nil {
+		return "", errors.New("create grb reviewer email template is nil")
+	}
```

Comment:
nit - can declare `b` after this check

```suggestion
	if sie.client.templates.systemIntakeCreateGRBReviewer == nil {
		return "", errors.New("create grb reviewer email template is nil")
	}
	var b bytes.Buffer
```

Reply:
Not sure if I should change this one file, change all the email files to do this, or just let it ride...

Reply:
let it ride 🏄